incubator/monochart/templates/_helpers.tpl

```diff
@@ -99,3 +99,32 @@ VolumeMounts template block for deployable resources
 {{- end }}
 {{- end -}}
 {{- end -}}
+
+
+{{/*
+Image template block
+*/}}
+{{- define "monochart.pod.image" -}}
+{{- $root := index . 0 -}}
+{{- $override := index . 1 -}}
+{{- $image := default $root.Values.image $override.image -}}
+image: {{ required "image.repository is required!" $image.repository }}:{{ required "image.tag is required!" $image.tag }}
+imagePullPolicy: {{ default "IfNotPresent" $image.pullPolicy }}
+{{- end -}}
+
+
+{{/*
+Template block for Pod's commands/args
+*/}}
+{{- define "monochart.pod.cmd" -}}
+{{- $root := index . 0 -}}
+{{- $propName := index . 1 -}}
+{{- $arr := index . 2 -}}
+{{- with $arr }}
+{{ $propName }}:
+{{- range $arg := . }}
+  - |-
+{{- tpl $arg $root | indent 4 -}}
```

Comment:
@alebabai, why we need, templatize commands?

Reply:
To be able to use values, release name, etc as it would be for the usual podTemplate.
There is an example that demonstrate possible usecase

Reply:
@alebabai not for commands. 
Commands should not be templates.
Only files in Configmaps and secrets (btw I'm not very optimistic about this, because it can make troubles with helmfile templating engine.)

Reply:
@goruha yes, i'm agree about helmfile templating engine, need to be tested (or find the way how to ignore template processing for the value).
But as for commands it's normal practice to template them.
For example postgres chart
https://github.com/helm/charts/blob/master/stable/postgresql/templates/statefulset.yaml#L67

Reply:
@alebabai 
In the changes, you can not do this https://github.com/helm/charts/blob/master/stable/postgresql/templates/statefulset.yaml#L65
Because it always would be `- |`

I prefer to do things that postgres do in this manner 

No need to templatize `commands`

```
statefulSet:
  default:
    enabled: true
    pod:
      commands:
        - "/scripts/entrypoint.sh"

configMaps:
  aws:
    enabled: true
    mountPath: /scripts
    files:
      entrypoint.sh: |-
             #!/bin/sh
            chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }} /bitnami
            if [ -d /bitnami/postgresql/data ]; then
              chmod  0700 /bitnami/postgresql/data;
            fi
```

Reply:
yea, maybe you're right